vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AnalysisTestBase.cs

```diff
@@ -110,8 +110,8 @@ protected async Task<(CodeAnalyzer, INamedTypeSymbol)> AnalyzeFileAsync(string c
         $@"{win10SdkFolder}\References\{win10SdkVersion}\Windows.Devices.Printers.PrintersContract\1.0.0.0\Windows.Devices.Printers.PrintersContract.winmd",
         $@"{win10SdkFolder}\References\{win10SdkVersion}\Windows.Devices.SmartCards.SmartCardBackgroundTriggerContract\3.0.0.0\Windows.Devices.SmartCards.SmartCardBackgroundTriggerContract.winmd",
         $@"{win10SdkFolder}\References\{win10SdkVersion}\Windows.Devices.SmartCards.SmartCardEmulatorContract\6.0.0.0\Windows.Devices.SmartCards.SmartCardEmulatorContract.winmd",
-        $@"{win10SdkFolder}\References\{win10SdkVersion}\Windows.Foundation.FoundationContract\3.0.0.0\Windows.Foundation.FoundationContract.winmd",
-        $@"{win10SdkFolder}\References\{win10SdkVersion}\Windows.Foundation.UniversalApiContract\8.0.0.0\Windows.Foundation.UniversalApiContract.winmd",
```

Comment:
these versions are fragile (they can be updated, and it makes it harder to update the target SDK version). Can the test be updated to discover which contract version is available in a given SDK?

Reply:
Filed: #7985

Reply:
Filed: #7985